packages/falso/src/tests/social.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { randSocial } from '../lib/social';
+
+describe('Social', () => {
+  it('Shuould return social object in formats [{name, link}]', () => {
+    const social = randSocial();
+
+    expect(social.name != '');
```

Comment:
This isn't valid test

Reply:
ok sorry, I've changed the test should I make another pull request?

Reply:
I've updated test
https://github.com/ngneat/falso/pull/124/commits/1e42b5446ebaf12e09c5b6a5d78f2d8b9161b20f